build-scripts/config_common.cmake

```diff
@@ -457,18 +457,22 @@ if (WAMR_BUILD_LINUX_PERF EQUAL 1)
   add_definitions (-DWASM_ENABLE_LINUX_PERF=1)
   message ("     Linux perf support enabled")
 endif ()
-if (NOT DEFINED WAMR_BUILD_QUICK_AOT_ENTRY)
-  # Enable quick aot/jit entries by default
-  set (WAMR_BUILD_QUICK_AOT_ENTRY 1)
-endif ()
-if (WAMR_BUILD_QUICK_AOT_ENTRY EQUAL 1)
-  add_definitions (-DWASM_ENABLE_QUICK_AOT_ENTRY=1)
-  message ("     Quick AOT/JIT entries enabled")
+if (WAMR_BUILD_AOT EQUAL 1 OR WAMR_BUILD_AOT EQUAL 1)
```

Comment:
Why need to check the condition twice by or?

Reply:
Should be WAMR_BUILD_JIT, fixed.